common/src/main/java/com/wynntils/models/items/annotators/gui/TerritoryAnnotator.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.annotators.gui;
+
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.item.ItemAnnotation;
+import com.wynntils.handlers.item.ItemAnnotator;
+import com.wynntils.models.items.items.gui.TerritoryItem;
+import com.wynntils.models.territories.type.GuildResource;
+import com.wynntils.models.territories.type.TerritoryStorage;
+import com.wynntils.models.territories.type.TerritoryUpgrade;
+import com.wynntils.utils.mc.LoreUtils;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public class TerritoryAnnotator implements ItemAnnotator {
```

Comment:
This can be a GuiItemAnnotator now

Reply:
Ah. I changed the item, I didn't know annotators were separate too..